instrumentation/graphql/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-graphql
 
+### v0.19.3 / 2022-05-05
+
+* (No significant changes)
```

Comment:
Not sure why this isn't listed https://github.com/open-telemetry/opentelemetry-ruby/pull/1213

Reply:
I believe the "chore" commits don't get included in the change log.